MobileWallet/Screens/RestoreWallet/RestoreWalletViewController.swift

```diff
@@ -113,6 +114,7 @@ extension RestoreWalletViewController: UITableViewDelegate, UITableViewDataSourc
                         self?.pendingView.hidePendingView { [weak self] in
                             UserDefaults.standard.set(true, forKey: HomeViewController.INTRO_TO_WALLET_USER_DEFAULTS_KEY)
```

Comment:
What's the reason for this being set in the restore process?

Reply:
we should not show background view with hand wave (intro screen) after restore wallet.

Reply:
Ok got thanks that makes sense 👍